docs/source/datasets.rst

```diff
@@ -110,6 +110,9 @@ click the link provided in the "Module" column.
 |                   |                     | - annotations: ✅   | - :ref:`technique`        |        |
 |                   |                     |                     | - :ref:`notes`            |        |
 +-------------------+---------------------+---------------------+---------------------------+--------+
+| classicalDB_      |  Tonality           | - audio: ❌         | - global :ref:`key`       | 881    |
```

Comment:
Are you using this instead of `tonality_classicalDB` because it's too long?

Reply:
yes